### PR TITLE
build: raw 11b with registry keys [draft for test]

### PR DESCRIPTION
**Pull request by ShiqianTao**
**What type of PR is this?**
/kind build

**What this PR does / why we need it**:

build: raw 11b with registry keys
- WS2019: CleanupReservedPorts, 652313229, 2059235981, 3767762061
- WS2022: PortExclusionChange, PortExclusionChange, CleanupReservedPorts, 527922829(DeltaHivePolicy=2), 2193453709, 3331554445

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -444,6 +444,31 @@ function Update-Registry {
             Write-Log "The current value of VfpNotReuseTcpOneWayFlowIsEnabled is $currentValue"
         }
         Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\VfpExt\Parameters" -Name VfpNotReuseTcpOneWayFlowIsEnabled -Value 1 -Type DWORD
+
+        Write-Log "Enable 4 fix in 2023-11B"
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:448]: fixes?

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -632,6 +657,55 @@ function Update-Registry {
             Write-Log "The current value of FwPerfImprovementChange is $currentValue"
         }
         Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\hns\State" -Name FwPerfImprovementChange -Value 1 -Type DWORD
+
+        Write-Log "Enable 7 fix in 2023-11B"
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:661]: fixes?

Comment (ShiqianTao) [reply vhdbuilder/packer/configure-windows-vhd.ps1]: thank you, updated

File: vhdbuilder/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (ShiqianTao) [reply vhdbuilder/packer/configure-windows-vhd.ps1]: thank you, updated

**Pull request closed without merging**